test/test_prototype_builtin_datasets.py

```diff
@@ -3,14 +3,25 @@
 
 import pytest
 import torch
-from builtin_dataset_mocks import parametrize_dataset_mocks, DATASET_MOCKS
+from builtin_dataset_mocks import parametrize_dataset_mocks, DATASET_MOCKS, TEST_HOME  # noqa: F401
 from torch.utils.data.datapipes.iter.grouping import ShardingFilterIterDataPipe as ShardingFilter
 from torch.utils.data.graph import traverse
 from torchdata.datapipes.iter import IterDataPipe, Shuffler
 from torchvision.prototype import transforms, datasets
 from torchvision.prototype.utils._internal import sequence_to_str
 
 
+@pytest.fixture(scope="session", autouse=True)
+def test_home(session_mocker, tmp_path_factory):
```

Comment:
This is a version as discussed yesterday where we the test does not need to take a `home` fixture to only pass it on to `dataset_mock.prepare`. 

A few things make the implementation a little complicated.

1. `pytest` only collects fixtures in `conftest.py` files or in the actual test module. Since the dataset mock module doesn't fall in either category, we need to keep the fixture here.
2. Since we now need to import `TEST_HOME` from another module, we need to make sure it is a pointer rather than a value. This means, we cannot `TEST_HOME = None` since after the import both values are decoupled. I've used a list here, but we can also have other objects.